files/wheelnav/update.json

```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "wheelnav",
+  "repo": "softwaretailoring/wheelnav",
+  "files": {
+    "include": ["wheelnav.js", "wheelnav.min.js", "wheelnav.min.js.map"]
```

Comment:
You need to add a `basePath` parameter to `js/dist` for this to work.
Also I dont think wheelnav.js is needed. The min file should be enough